ansible_role/tasks/main.yml

```diff
@@ -30,10 +16,52 @@
       The broker namespace must exist
   when: state == 'present'
 
-- include_tasks: "dao_{{ broker_dao_type }}.yaml"
+- name: Verify the broker has no outstanding service instances if deprovisioning
+  assert:
+    that:
+      - "lookup('k8s', kind='BundleInstance', namespace=broker_namespace, api_version='automationbroker.io/v1alpha1') | length == 0"
+    msg: |
+      The requested broker cannot be deprovisioned because it has outstanding service instances.
+      Please deprovision any services originally provisioned by this broker before deprovisioning the broker itself.
+  when: state == 'absent' and not broker_force_deprovision
+
+# Executed as an early task since the broker's configmap generation and creation
+# rely on this for the dashboard redirector
+- name: Create dashboard redirector route if OpenShift
+  k8s:
+    state: '{{ state }}'
+    definition: "{{ lookup('template', 'broker-redirector.route.yaml') | from_yaml }}"
+  when: ('route.openshift.io' in lookup('k8s', cluster_info='api_groups'))
+
+- name: Create broker config map
+  import_tasks: build_config.yaml
+
+- name: Print the generated broker config
+  debug: var=broker_config
+
+- include_tasks: "dao_{{ broker_config.dao.type }}.yaml"
 - include_tasks: "generate_certificate.yaml"
   when: ('route.openshift.io' not in lookup('k8s', cluster_info='api_groups')) | bool
 
+- name: 'Set broker config map state={{ state }}'
+  k8s:
+    state: '{{ state }}'
+    definition: '{{ broker_config_map }}'
+  register: configmap_result
+
+- name: Redeploy the broker if the configmap changed
```

Comment:
Were you able to get this working when you edited the CR?

Reply:
Yup, edited the CR with some new configmap vals, they propogate through and the operator bounces the pod. Works pretty well.